src/include/shell/shell.h

```diff
@@ -26,6 +26,18 @@
 #define DEF_COM 18
 #define HST_LEN 10
 
+#define RESET_POSIZ posiz = HST_LEN - 1;
```

Comment:
I don't think is a good idea to use macro that are abbreviation of italian words, please review macros and where possible also variable name (i.e. posiz should be pos, RESET_POSIZ should be RESET_POS etc...)